public/docs/release-notes.md

```diff
@@ -1,6 +1,69 @@
 Release Notes
 ===
 
+<i class="fa fa-tag"></i> 1.1.0-ce <i class="fa fa-clock-o"></i> 2018-03-28 15:00
+---
+
+### Security
+* Adding CSP headers
+* Prevent data-leak by wrong LDAP config
+* Generate dynamic sessionSecret if none is specifed
+
+### Enhancements
+* Add Minio support
+* Allow posting content to new notes by API
+* Add anonymous edit function in restricted mode
+* Add support for more Mimetypes on S3, minio and local filesystem uploads
+* Add basic cli tooling for local user management
+* Add referrer policy
+* Add more usable HTML5 tags
+* Add useridField in LDAP config
+* Add option for ReportURI for CSP violations
+* Add persistance for night mode
+* Allow setting sessionSecret by environment variable
+* Add night mode to features page
+
+### Fixes
+* Fix todo toggle function
+* Fix LDAP provider name usage
+* Fix errors on authenticated sessions for deleted users
+* Fix typo in database migration
+* Fix possible data truncation of authorship
+* Minor fixes in README.md
+* Allow usage of esc-key by codemirror
+* Fix array of emails in LDAP
+* Fix type errors by environment configs
+* Fix error message on some file API errors
+* Fix minor CSS issues in night mode
+
+### Refactors
+* Refactor contact
+* Refactor social media integration on main page
+* Refactor socket.io code to no longer use referrer
+* Refactor webpack config to need less dependencies in package.json
+* Refactor imageRouter for modularity
+* Refactor configs to be camel case
```

Comment:
I think either this should stand out more, or we need a "update to 1.1" explanation somewhere, like the [migration to ... things in the readme's upgrade section](https://github.com/hackmdio/hackmd#upgrade).

Reply:
Perhaps putting "breaking changes" at the top. I know this does not _break_ anything because of the way they are handled, but this _might_ break at some point in the far future.